question_box/questionapp/templates/base.html

```diff
@@ -31,7 +31,7 @@
              <button type="submit" class="btn btn-warning">Sign out</button>
              <a class="btn btn-success" href="/question/new" role="button">New Question &raquo;</a>
            </form>
-           <span class="navbar-right navbar-brand ">Welcome {{ request.user }}</span>
+           <span class="navbar-right navbar-brand ">Welcome <a href="/profile/{{ request.user.id }}">{{ request.user }}</a> <span class="badge">42 Points</span></span>
```

Comment:
So, right now the Welcome highlights when hovered. I'd say we either need to get rid of that, or just make the Welcome part of the link. 

Also, and we remove the coloring on the link? Turned the name blue. Think we should try to leave the styling like the template as much as possible.